src/Results/Results.js

```diff
@@ -2,16 +2,27 @@ import React, { Component } from 'react'
 
 class Results extends Component {
 	render() {
-		const {question, alternatives, votes} = this.props
+		const { question, alternatives, votes, open, users } = this.props
+
+		const numVotes = votes.reduce((a, b) => a + b, 0);
+		// user.attending is a boolean
+		// However, true + true = 2 (and so on) in Javascript, so the map is actually unnecessary
+		const attending = users.map(u => u.attending ? 1 : 0).reduce((a, b) => a + b, 0)
```

Comment:
Skulle nog byta den till `users.filter(user => user.attending).length`

Reply:
Efter `filter` kommer vi då ha en lista med `user`-objekt istället för en lista med 1:or och 0:or, men kan lösas såhär
`users.filter(u => u.attending).reduce((a, b) => a + b.attending, 0)`

Reply:
Fixed in b4ecea8